### ISSUE TITLE
[A11Y][List item] Trailing, labels and leading elements should be grouped

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#919]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone

### OS Version

26

### App version

3.0.0(273)

### Describe the issue

Leading, labels and trailing elements are not grouped

### Expected Behavior

Group leading, labels, trailing elements (in that order) so screen reader reads everything at once.

**Comment by pya35** [comment]
fixed on build 729
